pypit/arcimage.py

```diff
@@ -71,30 +71,45 @@ def __init__(self, file_list=[], spectrograph=None, settings=None, det=1, setup=
         self.setup = setup
 
         # Start us up
-        processimages.ProcessImages.__init__(self, file_list, spectrograph=spectrograph, settings=settings, det=det)
+        processimages.ProcessImages.__init__(self, file_list, spectrograph=spectrograph, settings=settings, user_settings = user_settings, det=det)
 
         # Attributes (set after init)
         self.frametype = frametype
 
         # Settings
         # The copy allows up to update settings with user settings without changing the original
-        if settings is None:
-            # Defaults
-            self.settings = processimages.default_settings()
-        else:
-            self.settings = settings.copy()
-            # The following is somewhat kludgy and the current way we do settings may
-            #   not touch all the options (not sure .update() would help)
-            if 'combine' not in settings.keys():
-                if self.frametype in settings.keys():
-                    self.settings['combine'] = settings[self.frametype]['combine']
+
+
+        # TODO, JFH I think the logic below flawed here. The processimages class is already dealing with the settings and setting them to defaults if the user
```

Comment:
This is all being revamped in the new settings anyhow.

Reply:
Okay let's get this stuff into master ASAP. I've written a quicklook pipeline for NIRES based on PYPIT, and it is painful when things keep changing.